src/main/java/es/thalesalv/gptbot/adapters/rest/OpenAIApiService.java

```diff
@@ -24,14 +31,21 @@ public class OpenAIApiService {
     @Value("${config.openai.api-token}")
     private String openAiToken;
 
+    private final JDA jda;
     private final WebClient webClient;
+    private final ContextDatastore contextDatastore;
 
     private static final String BEARER = "Bearer ";
     private static final String OPENAI_API_BASE_URL = "https://api.openai.com";
+    private static final String UNKNOWN_ERROR = "An error occured and the message could not be sent to the model. Your message will be removed. Please try again.\n**Message content:** {0}";
+    private static final String EMPTY_RESPONSE = "The model did not generate an output due to a problem. Please try again. Your message will be removed.\n**Message content:** {0}";
```

Comment:
Do we really want to remove the message in this case when someone might just retrigger it by tagging the bot if there is an error on the OpenAI end of things?

Reply:
The tagging the bot thing only works for RPG, the normal conversation rooms don't have that feature. Should we add it?

Reply:
But it's nice to have some feedback if something went wrong

Reply:
> The tagging the bot thing only works for RPG, the normal conversation rooms don't have that feature. Should we add it?

I think so? I mean I certainly use it in non-rpg channels when a completion gets dumped by OAI.